pkg/tuf_v2/repository/map.json

```diff
@@ -0,0 +1,13 @@
+{
+    "repositories": {
+      "public-good-instance": ["https://storage.googleapis.com/sigstore-tuf-root/"]
```

Comment:
Could we add another entry to see how this might look for say public and private Sigstore instance. I think that would also help drive my understanding on how the 'paths' below would be configured (as an example for testing cases where the local TUF remote is at http://tuf.tuf-system.svc in scaffolding) so that the right things would happen.
I'd be happy to start testing these out with the policy-controller btw if that would help validate some of these use cases. Naive guess is that "paths" section for a custom one would list the URLs of Fulcio, Rekor, CTLog, etc. but again, I think an example would help here.